### DISCUSSION TITLE
cd command error on vck board

### DISCUSSION THREAD

**Discussion by freud96** [bol-edu/HLS-SOC-Discussions#47] [Q&A]
Hello,

I entered vck board platform on onlineFPGA, when I use the cd command, it gives error as follow:
05.JQoMCD@HLS05:~$ cd LabC
/mnt/HLSNAS/05.JQoMCD/.changedir.sh: 1: pushd: not found

is there any solution that can help solve this?

**Comment by kevinjantw** [comment]
Hi,
The system issue was solved. You can re-login and work correctly.
